src/components/ModelTableList/CloudGroup.js

```diff
@@ -175,5 +175,5 @@ export default function CloudGroup({ filters }) {
       />
     );
   }
-  return <div className="cloud-group u-overflow--scroll">{cloudTables}</div>;
```

Comment:
Remove the scrollbar in no mac interface when it's not needed:
![image](https://user-images.githubusercontent.com/36013798/195399185-ea89a3da-c0ec-4469-9e46-a8b72e590875.png)